annotations.py

```diff
@@ -22,9 +22,13 @@
 from calibre_plugins.annotations.common_utils import Logger
 from calibre_plugins.annotations.config import plugin_prefs
 
+try:
+    from PyQt5.QtGui import QColor
+except ImportError:
+    from PyQt4.QtGui import QColor
```

Comment:
Is this support for different Python versions used by Calibre now or in the future?

Reply:
Right, this is how all Qt dependencies are imported in this plugin, not just here.